config/default/config_split.config_split.sitenow_webform_ais_rfi.yml

```diff
@@ -12,7 +12,9 @@ storage: folder
 folder: ../config/features/sitenow_webform_ais_rfi
 module:
   sitenow_webform_ais_rfi: 0
+  uiowa_maui: 0
```

Comment:
Is this here because the module is required? If so, it might be better to make it a dependency of `sitenow_webform_ais_rfi`, which will also ensure it is enabled

Reply:
yes, the module is required because it leverages the API calls in that module. I added the module to the info file, but for existing sites with the feature on but uiowa_maui disabled, it didn't enable it.

i can remove it from here but i think an update hook would be needed instead for education, others? that have this feature turned on already

Reply:
i don't know, i think it needs to stay in the split. otherwise it is trying to disable it since it isn't in core.extension default. The update hook fires and enables the module, 

```php
/**
 * Install uiowa_maui for existing websites with this feature enabled.
 */
function sitenow_webform_ais_rfi_update_10001() {
  // This module is now a dependency.
  \Drupal::service('module_installer')->install(['uiowa_maui']);
}
```

But then I get:

```
The import failed due to the following reasons:                                              
  Unable to uninstall the <em class="placeholder">uiowa_maui</em> module since the <em class=  
  "placeholder">SiteNow Webform AIS RFI</em> module is installed.  
```

Reply:
For example: https://github.com/uiowa/uiowa/blob/cca204e01569afa23cb142cad6124f2348478164/config/default/config_split.config_split.thesis_defense.yml#L14

Reply:
Nevermind then. Not trying to blow this up. Just noting that while this is an established pattern, it creates complications. If we ever introduce config associated with the `uiowa_maui` module, then this will force that config to be completely split for this feature split. We have previously moved low cost dependencies into our default config to avoid just this type of thing. I was just trying to think about it in a future context.

Reply:
I keep having to remind myself that uiowa_maui isn't turned on by default. I think we are getting closer. Maybe it could be on but the academic dates block has a separate permission or toggle?

Reply:
https://github.com/uiowa/uiowa/issues/8479#issuecomment-2675530401 seems to suggest that these options should be available for all sites?

Reply:
Before Brian left, we were talking about this and how their specific requirements is reducing the usefulness of it for all. https://github.com/uiowa/uiowa/issues/8479#issuecomment-2707012788

Reply:
Got more confirmation of this today from Ali that the needs right now are very undergrad specific and would not be particularly useful for others at this time.

I'd consider this an early prototype version. Depending on where it goes it could be updated for the websites using it or it could become deprecated and forms are updated with a replacement.